bitnami/mxnet/README.md

```diff
@@ -0,0 +1,262 @@
+| `image.registry`                     | MXNet image registry                                                                       | `docker.io`                                             |
+| `image.repository`                   | MXNet image name                                                                           | `bitnami/MXNet`                                       |
+| `image.tag`                          | MXNet image tag                                                                            | `{VERSION}`                                             |
+| `image.pullPolicy`                   | Image pull policy                                                                            | `IfNotPresent`                                          |
+| `image.pullSecrets`                  | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `image.debug`                        | Specify if debug logs should be enabled                                                      | `false`                                                 |
+| `git.registry`                       | Git image registry                                                                           | `docker.io`                                             |
+| `git.repository`                     | Git image name                                                                               | `bitnami/git`                                           |
+| `git.tag`                            | Git image tag                                                                                | `latest`                                                |
+| `git.pullPolicy`                     | Git image pull policy                                                                        | `Always`                                                |
+| `git.pullSecrets`                    | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `service.type`                         | Kubernetes service type                                                                      | `ClusterIP`                                             |
+| `entrypoint.file`                    | Main entrypoint to your application. If not speficied, it will be a `sleep infinity` command                                                          | `''`                                                    |
+| `entrypoint.args`                    | Args required by your entrypoint                                                             | `nil`                                                   |
+| `entrypoint.workDir`                    | Working directory for launching the entrypoint                                                          | `'/app'`                                                    |
```

Comment:
Can you indent the table? There is a plugin for vscode that does it automatically (Markdown All in one)

Reply:
Done